src/Model.js

```diff
@@ -368,6 +382,20 @@ class Model {
       { relationships, attributes }
     ))
 
+    if (relationships) {
```

Comment:
I think we should just keep an eye out for this when we are testing this in `portal`. No change needed, more of a note for us to watch out if something blows up.